zebra-test/Cargo.toml

```diff
@@ -21,6 +21,7 @@ regex = "1.3.9"
 thiserror = "1.0.20"
 tempdir = "0.3.7"
 pretty_assertions = "0.6.1"
+owo-colors = "1.1.3"
```

Comment:
owo whats this?  https://crates.io/crates/owo-colors

Reply:
looks neat, no objection to adding this to test code

Reply:
> owo whats this?

:rofl: